sdks/java/io/xml/build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
   shadow library.java.stax2_api
   shadow library.java.findbugs_jsr305
   shadow library.java.woodstox_core_asl
+  shadow library.java.jaxb_api
```

Comment:
`shadowTest` to match the same intended scope inserted in the pom.xml?

Reply:
I was also thinking about `testRuntime`, but `shadowTest` seems like right dependency type to consistent with intended scope also provided in sdks/java/io/xml/pom.xml.